x-pack/plugins/security_solution/public/exceptions/pages/shared_lists/index.tsx

```diff
@@ -329,44 +330,28 @@ export const SharedLists = React.memo(() => {
       iconSide="right"
       onClick={onRowSizeButtonClick}
     >
-      {/* TODO move to translations */}
-      {`Rows per page: ${rowSize}`}
+      {i18n.allExceptionsRowPerPage(rowSize)}
     </EuiButtonEmpty>
   );
 
   const getIconType = (size: number) => {
     return size === rowSize ? 'check' : 'empty';
   };
 
+  const onPerPageClick = useCallback((size: number) => {
+    closeRowSizePopover();
+    setRowSize(size);
+    setActivePage(0);
```

Comment:
This is where the fix is. There's some jank with the page, reloading when I don't think we need to on row selection, but for now just fixing this bug.